src/store/reducers/jugglingTricksSlice.js

```diff
@@ -1,30 +1,42 @@
 import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { API_URL, HTTP_STATUS } from "../../app/constants";
 import threeBallTricks from "../../assets/json/jugglingTricks/ThreeBallTricks.json";
 import fourBallTricks from "../../assets/json/jugglingTricks/FourBallTricks.json";
-import { API_URL, HTTP_STATUS } from "../../app/constants";
+import fiveBallTricks from "../../assets/json/jugglingTricks/FiveBallTricks.json";
```

Comment:
I think we should move towards having one trick DB/slice, in that all the tricks live in one table and we can fetch them with one call instead of having separate calls for each?

Reply:
I agree that we should move in the direction- This was put in just to test the new design. Would probably be a good next step to work towards this.